v21.2/create-changefeed.md

```diff
@@ -102,20 +123,72 @@ Option | Value | Description
+`topic_in_value`         | [`BOOL`](bool.html)              | <span class="version-tag">New in v21.2:</span> Set to include the topic in each emitted row update. Note this is automatically set for [webhook sinks](#webhook-sink).
+`webhook_sink_config`    | [`STRING`](string.html)          | <span class="version-tag">New in v21.2:</span> Set fields to configure sink batching and retries. The schema is as follows:<br><br> `{ "Flush": { "Messages": ..., "Bytes": ..., "Frequency": ..., }, "Retry": {"Max": ..., "Backoff": ..., } }`. <br><br>**Note** that if either `Messages` or `Bytes` are nonzero, then a non-zero value for `Frequency` must be provided. <br><br>See the [Webhook sink configuration section](#webhook-sink-configuration) for more details on using this option.
 
 {{site.data.alerts.callout_info}}
  Using the `format=experimental_avro`, `envelope=key_only`, and `updated` options together is rejected. `envelope=key_only` prevents any rows with updated fields from being emitted, which makes the `updated` option meaningless.
 {{site.data.alerts.end}}
 
+#### Webhook sink configuration
+
+<span class="version-tag">New in v21.2:</span> The `webhook_sink_config` option allows the changefeed flushing and retry behavior to your webhook sink to be configured.
+
+The following details the configurable fields:
+
+* `Flush.Messages`: When the batch reaches this configured size, it should be flushed (batch sent).
+  * Type, `INT`. Default, `0`.
+* `Flush.Bytes`: When the total byte size of all the messages in the batch reaches this amount, it should be flushed.
+  * Type, `INT`. Default, `0`.
+* `Flush.Frequency`: When this amount of time has passed since the **first** received message in the batch without it flushing, it should be flushed.
+  * Type, `INTERVAL`. Default, `"0s"`.
+* `Retry.Max`: The maximum amount of time the sink will retry a single HTTP request to send a batch. This value must be positive (> 0). If infinite retries are desired, use `inf`.
+  * Type, `INT` or `STRING`. Default, `"0s"`.
+* `Retry.Backoff`: The initial backoff the sink will wait after the first failure. The backoff will double (exponential backoff strategy), until the max is hit.   
+  * Type, `INTERVAL`. Default, `"500ms"`.
+
+{{site.data.alerts.callout_info}}
+Setting either `Messages` or `Bytes` with a non-zero value without setting `Frequency`, will cause the sink to assume `Frequency` has an infinity value. This configuration would be invalid and will cause an error, since the messages could sit in a batch indefinitely if the other conditions do not trigger. Therefore, if either `Messages` or `Bytes` have a non-zero value, then a non-zero value for `Frequency` **must** be provided.
+{{site.data.alerts.end}}
+
+Some complexities to consider when setting `Flush` fields for batching:
```

Comment:
the colon at the end of this makes me think this wants to be a bulleted list of things?  IIRC the code blocks can be indented to work inside bulleted lists